tests/test_data.py

```diff
@@ -390,7 +411,10 @@ def test_sample_sizes_and_types(
                 sample = ds.sample(n_samples)
                 assert isinstance(sample, type(trans))
                 for k, v in dataclasses.asdict(sample).items():
-                    trans_v: np.ndarray = trans_dict[k]
+                    if k == "infos":
```

Comment:
Could be simplified:

```python
trans_v = trans_dict[k]
if k == "info":
  trans_v = np.array(trans_v)
assert isinstance(trans_v, np.ndarray)
```

Reply:
I did this if-else structure to avoid a pytype error. I'll try again to see if it's okay now.